apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/TemplateRenderer.swift

```diff
@@ -357,9 +364,8 @@ struct ImportStatementTemplate {
 struct ModuleImportStatementTemplate {
 
   static func template(
-    moduleImports: OrderedSet<String>?
-  ) -> TemplateString? {
-    guard let moduleImports else  { return nil }
+    moduleImports: OrderedSet<String>
+  ) -> TemplateString {
     return """
     \(moduleImports.map { "import \($0)" }.joined(separator: "\n"))
```

Comment:
Should we have a comment above these to identify these as custom configured imports?